revproxy/views.py

```diff
@@ -38,30 +45,57 @@ def __init__(self, *args, **kwargs):
 
     @property
     def upstream(self):
+        """This method is not implemanted on the base class """
         raise NotImplementedError('Upstream server must be set')
 
     @property
     def diazo_rules(self):
```

Comment:
This is actually the default for diazo rules. As this is a property it can be overwritten by a class attribute.